tensorflow/lite/kernels/internal/compatibility.h

```diff
@@ -83,6 +83,8 @@ using int16 = std::int16_t;
 using uint16 = std::uint16_t;
 using int32 = std::int32_t;
 using uint32 = std::uint32_t;
+using int64 = std::int64_t;
+using uint64 = std::uint64_t;
```

Comment:
`In file included from ./knowledge/cerebra/sense/text_classifier/training/augmenter.h:9:
./knowledge/cerebra/sense/text_classifier/utils/random-source.h:38:27: error: reference to 'int64' is ambiguous
  explicit MtRandomSource(int64 seed);
                          ^
./third_party/tensorflow/lite/kernels/internal/compatibility.h:86:7: note: candidate found by name lookup is 'int64'
using int64 = std::int64_t;
      ^
third_party/absl/base/integral_types.h:75:29: note: candidate found by name lookup is 'int64'
typedef long long           int64;
                            ^
1 error generated.

In file included from knowledge/cerebra/sense/text_classifier/training/augmenter.cc:1:
In file included from ./knowledge/cerebra/sense/text_classifier/training/augmenter.h:9:
./knowledge/cerebra/sense/text_classifier/utils/random-source.h:38:27: error: reference to 'int64' is ambiguous
  explicit MtRandomSource(int64 seed);
                          ^
./third_party/tensorflow/lite/kernels/internal/compatibility.h:86:7: note: candidate found by name lookup is 'int64'
using int64 = std::int64_t;
      ^
third_party/absl/base/integral_types.h:75:29: note: candidate found by name lookup is 'int64'
typedef long long           int64;
                            ^`

Reply:
please check above error ,this is failing internally.

Reply:
removed ambiguous reference. please have a look.